doc/operator-scope.md

```diff
@@ -1,9 +1,31 @@
-# Operator SDK: Operator Scope
+# Operators and CRD scope with Operator SDK
+
+- [Namespace-scoped operator usage](#namespace-scoped-operator-usage)
+- [Cluster-scoped operator usage](#cluster-scoped-operator-usage)
+  - [Changes required in the scaffold project for the cluster-scoped operator](#changes-required-in-the-scaffold-project-for-the-cluster-scoped-operator)
+  - [Example for cluster scoped operator](#example-for-cluster-scoped-operator)
+- [CRD scope](#crd-scope)
+  - [CRD cluster-scoped usage](#crd-cluster-scoped-usage)
+  - [Example for change the CRD scope from namespace to cluster](#example-for-change-the-crd-scope-from-namespace-to-cluster)
+
+## Overview
 
 A namespace-scoped operator watches and manages resources in a single namespace, whereas a cluster-scoped operator watches and manages resources cluster-wide. Namespace-scoped operators are preferred because of their flexibility. They enable decoupled upgrades, namespace isolation for failures and monitoring, and differing API definitions.
 
 However, there are use cases where a cluster-scoped operator may make sense. For example, the [cert-manager](https://github.com/jetstack/cert-manager) operator is often deployed with cluster-scoped permissions and watches so that it can manage issuing certificates for an entire cluster.
 
+## Namespace-scoped operator usage
+
+This scope is ideal for operator projects which will control resources just in one namespace, which is where the operator is deployed.
+
+> **NOTE**: It is the default configuration for the initial projects created by operator-sdk which means that it will NOT have a `Cluster Role` defined in the `deploy/role_binding.yaml`.  
```

Comment:
```suggestion
> **NOTE:** Initial projects created by `operator-sdk` are namespace-scoped by default which means that it will NOT have a ClusterRole defined in the deploy/role_binding.yaml"
```